.nvmrc

```
@@ -0,0 +1 @@
+v20.18.1
```

Version 20 is enough, no improvement into specify the minor too.

```suggestion
v20
```

---

Maybe we can use v22?

---

+1 for 22